app/routes/events/components/EventFooter.tsx

```diff
@@ -39,8 +38,8 @@ const EventFooter = ({ icalToken }: Props) => (
     {icalToken && (
       <p className={styles.section}>
         Her kan du importere arrangementer og møter til din favorittkalender!
-        For innstillinger se
-        <Link to="/users/me/settings"> her</Link>.
+        <br />
+        Trykk på en av lenkene under for å legge inn i din kalender.
```

Comment:
Is this comment really needed? I kinda feel like it's smør på flesk, but I'll let you decide😄

Reply:
➕